README.adoc

```diff
@@ -409,7 +409,7 @@ InventoryManager.java
 include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[]
 ----
 
-Annotate the [hotspot=addMethod file=0]`add()` and [hotspot=listMethod file=0]`list()` methods with the [hotspot=addWithSpan hotspot=listWithSpan file=0]`@WithSpan` annotation. You can annotate the [hotspot=spanAttribute file=0]`host` parameter with the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation with a customized value to indicate that it is part of the trace.
+The [hotspot=addMethod file=0]`add()` and [hotspot=listMethod file=0]`list()` methods are annotated with the [hotspot=addWithSpan hotspot=listWithSpan file=0]`@WithSpan` annotation. The OpenTelemetry instrumentation provides a new span for each method. You can now collect and trace the spans across different services. After creating a span, you have the option to include desired parameters with their values in the span by using the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation. For example, the [hotspot=spanAttribute file=0]`host` parameter can be assigned [hotspot=spanAttribute file=0]`hostname` as its attribute name using the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation, which gives the benefit of tracing the parameters within the [hotspot=addMethod file=0]`add` span.
```

Comment:
```suggestion
The [hotspot=addMethod file=0]`add()` and [hotspot=listMethod file=0]`list()` methods are annotated with the [hotspot=addWithSpan hotspot=listWithSpan file=0]`@WithSpan` annotation. The OpenTelemetry instrumentation provides a new span for each method. You can now collect and trace the spans across different services. After creating a span, you have the option to include desired parameters with their values in the span by using the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation. For example, the [hotspot=spanAttribute file=0]`host` parameter is assigned [hotspot=spanAttribute file=0]`hostname` as its attribute name using the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation, which gives the benefit of tracing the parameters within the [hotspot=addMethod file=0]`add` span.
```

The example is what is shown in the guide app code (you're not instructing the reader that this is something they _can_ do if they want; it's already been done for them to show them how it can be done).